### ISSUE TITLE
Open new blank tab as sibling of current tab not repected for parent tabs

### ISSUE THREAD

**Issue by bensowden** [piroor/treestyletab#1606]
## Short description

When opening new blank tab using cmd+t with the setting set to open as sibling of current tab the new tab opens as the next sibling of the current tab for top level parent tabs (it does not happen for tabs without children or for child tabs with further children) rather than at the bottom of the list as expected.

## Steps to reproduce

 1. Start Firefox with clean profile.
 2. Install TST.
 3. Set the 'Open new blank tab as' setting to 'sibling of the current tab'
 4. Open a series of tabs that looks like:
```
A (selected)
\- B (either collapsed or expanded)
C
```
 5. With tab A in focus open a new tab

## Expected result

The new tab (D) should open at the bottom of the list at the same level as A (top level after C).
```
A
\- B (either collapsed or expanded)
C
D (selected)
```

## Actual result

The new tab (D) opens are the correct level but between B and C
```
A
\- B (either collapsed or expanded)
D (selected)
C
```

## Environment

 * Platform (OS): MacOS 10.13
 * Version of Firefox: 57.0
 * Version (or revision) of Tree Style Tab: 2.2.11

**Comment by piroor** [comment]
Thanks. 5b80e67 should fix this.

**Comment by bensowden** [comment]
I can confirm this is fixed for me in v2.3.0 - this issue can be closed. Thanks for the quick fix!
